syne_tune/optimizer/schedulers/hyperband.py

```diff
@@ -48,10 +52,22 @@
 logger = logging.getLogger(__name__)
 
 
+RUNG_SYSTEMS = {
+    "stopping": StoppingRungSystem,
+    "promotion": PromotionRungSystem,
+    "pasha": PASHARungSystem,
+    "rush_promotion": RUSHPromotionRungSystem,
+    "rush_stopping": RUSHStoppingRungSystem,
+    "cost_promotion": CostPromotionRungSystem,
+    "dyhpo": DyHPORungSystem,
+}
+
+
 _ARGUMENT_KEYS = {
     "resource_attr",
     "grace_period",
     "reduction_factor",
+    "rung_increment",
```

Comment:
With rung_increment, I am also allowing for linear spacing (which is what DyHPO uses). If reduction_factor is given, that overrules rung_increment,